.github/workflows/check-pr.yml

```
@@ -14,4 +14,4 @@ jobs:
           java-version: '11'
           cache: 'sbt'
       - name: Check PR
-        run: sbt --batch "compile"
+        run: sbt --batch ";compile;test"
```

leading `;` is not required:
```suggestion
        run: sbt --batch "compile;test"
```
see the first and last highlighted lines in https://github.com/sbt/sbt/pull/4456/files#diff-9995628b4609b3ce017a864e4647d00ffd4aef454fa6766fa40f6c356cc75c1eR26-R54